<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>aplMainWindow</class>
 <widget class="QMainWindow" name="aplMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1265</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QComboBox" name="pathBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2" colspan="3">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="childrenCollapsible">
          <bool>false</bool>
         </property>
         <widget class="QTreeView" name="directoryTree">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>800</height>
           </size>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QTabWidget" name="browser">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>0</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>30</height>
           </size>
          </property>
          <property name="elideMode">
           <enum>Qt::ElideNone</enum>
          </property>
          <property name="tabsClosable">
           <bool>true</bool>
          </property>
          <property name="movable">
           <bool>true</bool>
          </property>
          <property name="tabBarAutoHide">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="home">
           <attribute name="icon">
            <iconset>
             <normaloff>:/icon/default theme/home-.png</normaloff>:/icon/default theme/home-.png</iconset>
           </attribute>
           <attribute name="title">
            <string>Home</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QWidget" name="scaned" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>600</width>
                <height>234</height>
               </size>
              </property>
              <property name="focusPolicy">
               <enum>Qt::StrongFocus</enum>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="1" column="0">
                <widget class="QPushButton" name="btn_doc">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::StrongFocus</enum>
                 </property>
                 <property name="text">
                  <string>   文档</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>:/icon/default theme/file-pdf-fill.png</normaloff>:/icon/default theme/file-pdf-fill.png</iconset>
                 </property>
                 <property name="autoDefault">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QPushButton" name="btn_collect">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::StrongFocus</enum>
                 </property>
                 <property name="text">
                  <string>   收藏</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>:/icon/default theme/folder-info-fill.png</normaloff>:/icon/default theme/folder-info-fill.png</iconset>
                 </property>
                 <property name="autoDefault">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="btn_video">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::StrongFocus</enum>
                 </property>
                 <property name="text">
                  <string> 视频</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>:/icon/default theme/video-gallery.png</normaloff>:/icon/default theme/video-gallery.png</iconset>
                 </property>
                 <property name="autoDefault">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QPushButton" name="btn_picture">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::StrongFocus</enum>
                 </property>
                 <property name="text">
                  <string>图片</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>:/icon/default theme/photo.png</normaloff>:/icon/default theme/photo.png</iconset>
                 </property>
                 <property name="autoDefault">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="btn_zip">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::StrongFocus</enum>
                 </property>
                 <property name="text">
                  <string> 压缩文件</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>:/icon/default theme/folder-zip-fill.png</normaloff>:/icon/default theme/folder-zip-fill.png</iconset>
                 </property>
                 <property name="autoDefault">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QPushButton" name="btn_music">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::StrongFocus</enum>
                 </property>
                 <property name="text">
                  <string> 音乐</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>:/icon/default theme/folder-music-fill.png</normaloff>:/icon/default theme/folder-music-fill.png</iconset>
                 </property>
                 <property name="autoDefault">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>index</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab">
           <attribute name="icon">
            <iconset>
             <normaloff>:/icon/default theme/add.png</normaloff>:/icon/default theme/add.png</iconset>
           </attribute>
           <attribute name="title">
            <string/>
           </attribute>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="searchBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>34</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>290</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>290</width>
        <height>50</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="btn_backward">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btn_forward">
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btn_backspace">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>35</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionnewFile"/>
   <addaction name="actionhome"/>
   <addaction name="separator"/>
   <addaction name="actioncut"/>
   <addaction name="actioncopy"/>
   <addaction name="actionpaste"/>
   <addaction name="actionshare"/>
   <addaction name="actiondelete"/>
   <addaction name="separator"/>
   <addaction name="actionsetting"/>
  </widget>
  <action name="actionnewFile">
   <property name="icon">
    <iconset>
     <normaloff>:/icon/default theme/file-add-line.png</normaloff>:/icon/default theme/file-add-line.png</iconset>
   </property>
   <property name="text">
    <string>newFile</string>
   </property>
   <property name="toolTip">
    <string>新建文件夹</string>
   </property>
  </action>
  <action name="actionforward">
   <property name="icon">
    <iconset>
     <normaloff>:/icon/default theme/forward-.png</normaloff>:/icon/default theme/forward-.png</iconset>
   </property>
   <property name="text">
    <string>forward</string>
   </property>
   <property name="toolTip">
    <string>前进</string>
   </property>
  </action>
  <action name="actionbackward">
   <property name="icon">
    <iconset>
     <normaloff>:/icon/default theme/back-arrow-.png</normaloff>:/icon/default theme/back-arrow-.png</iconset>
   </property>
   <property name="text">
    <string>backward</string>
   </property>
   <property name="toolTip">
    <string>后退</string>
   </property>
  </action>
  <action name="actionhome">
   <property name="icon">
    <iconset>
     <normaloff>:/icon/default theme/home-.png</normaloff>:/icon/default theme/home-.png</iconset>
   </property>
   <property name="text">
    <string>home</string>
   </property>
   <property name="toolTip">
    <string>回到主页</string>
   </property>
  </action>
  <action name="actioncopy">
   <property name="icon">
    <iconset>
     <normaloff>:/icon/default theme/file-copy-2-fill.png</normaloff>:/icon/default theme/file-copy-2-fill.png</iconset>
   </property>
   <property name="text">
    <string>copy</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
  </action>
  <action name="actionpaste">
   <property name="icon">
    <iconset>
     <normaloff>:/icon/default theme/sticker-.png</normaloff>:/icon/default theme/sticker-.png</iconset>
   </property>
   <property name="text">
    <string>paste</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
  </action>
  <action name="actioncut">
   <property name="icon">
    <iconset>
     <normaloff>:/icon/default theme/cut.png</normaloff>:/icon/default theme/cut.png</iconset>
   </property>
   <property name="text">
    <string>cut</string>
   </property>
   <property name="toolTip">
    <string>剪切</string>
   </property>
  </action>
  <action name="actionshare">
   <property name="icon">
    <iconset>
     <normaloff>:/icon/default theme/share-.png</normaloff>:/icon/default theme/share-.png</iconset>
   </property>
   <property name="text">
    <string>share</string>
   </property>
   <property name="toolTip">
    <string>分享</string>
   </property>
  </action>
  <action name="actiondelete">
   <property name="icon">
    <iconset>
     <normaloff>:/icon/default theme/delete.png</normaloff>:/icon/default theme/delete.png</iconset>
   </property>
   <property name="text">
    <string>delete</string>
   </property>
   <property name="toolTip">
    <string>删除</string>
   </property>
  </action>
  <action name="actionempty">
   <property name="text">
    <string/>
   </property>
   <property name="toolTip">
    <string>空的占位置</string>
   </property>
  </action>
  <action name="actionrefresh">
   <property name="icon">
    <iconset>
     <normaloff>:/icon/default theme/rotate-right.png</normaloff>:/icon/default theme/rotate-right.png</iconset>
   </property>
   <property name="text">
    <string>refresh</string>
   </property>
   <property name="toolTip">
    <string>刷新</string>
   </property>
  </action>
  <action name="actionlayout">
   <property name="icon">
    <iconset>
     <normaloff>:/icon/default theme/grid.png</normaloff>:/icon/default theme/grid.png</iconset>
   </property>
   <property name="text">
    <string>layout</string>
   </property>
   <property name="toolTip">
    <string>布局</string>
   </property>
  </action>
  <action name="actionsetting">
   <property name="icon">
    <iconset>
     <normaloff>:/icon/default theme/settings-.png</normaloff>:/icon/default theme/settings-.png</iconset>
   </property>
   <property name="text">
    <string>setting</string>
   </property>
   <property name="toolTip">
    <string>设置</string>
   </property>
  </action>
  <action name="actionbigIcon">
   <property name="icon">
    <iconset>
     <normaloff>:/icon/default theme/grid-.png</normaloff>:/icon/default theme/grid-.png</iconset>
   </property>
   <property name="text">
    <string>大图标</string>
   </property>
   <property name="toolTip">
    <string>大图标</string>
   </property>
  </action>
  <action name="actionsmallIcon">
   <property name="icon">
    <iconset>
     <normaloff>:/icon/default theme/grid-_1.png</normaloff>:/icon/default theme/grid-_1.png</iconset>
   </property>
   <property name="text">
    <string>小图标</string>
   </property>
   <property name="toolTip">
    <string>小图标</string>
   </property>
  </action>
  <action name="actiondetails">
   <property name="icon">
    <iconset>
     <normaloff>:/icon/default theme/layout-.png</normaloff>:/icon/default theme/layout-.png</iconset>
   </property>
   <property name="text">
    <string>详细内容</string>
   </property>
   <property name="toolTip">
    <string>详细内容</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
